code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -10,10 +10,10 @@
 	name = "Cough"
 	desc = "The virus irritates the throat of the host, causing occasional coughing. Each cough will try to infect bystanders who are within 1 tile of the host with the virus."
 	illness = "Jest Infection"
-	stealth = -1
-	resistance = 3
-	stage_speed = 1
-	transmittable = 2
+	stealth = 50
+	resistance = 50
+	stage_speed = 50
+	transmittable = 50
```

Comment:
?????

Reply:
that would be my debug stuff, this makes second time i forget to remove them!

Reply:
i try to test every function that i can.
birb brain.